From: Michael Biebl <biebl@debian.org>
Date: Sat, 27 Jun 2020 12:17:31 +0200
Subject: Revert "change systemd service file to wait for network"

The default rsyslog configuration in Debian uses imuxsock which relies
on ForwardToSyslog to work properly. This commit broke that resulting in
syslog.service and syslog.socket no longer being started correctly and
rsyslog taking over /dev/log from journald.

As for the remaining bits:
We don't enable remote syslog by default, so we don't need/want to
wait for network-online.target during boot.

This reverts commit 7a671da74f78047b3fd0eb65203f045de28cec0f.

Closes: #963715
---
 rsyslog.service.in | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/rsyslog.service.in b/rsyslog.service.in
index 839765d..b761ae5 100644
--- a/rsyslog.service.in
+++ b/rsyslog.service.in
@@ -1,10 +1,8 @@
 [Unit]
 Description=System Logging Service
-;Requires=syslog.socket
-Wants=network.target network-online.target
-After=network.target network-online.target
+Requires=syslog.socket
 Documentation=man:rsyslogd(8)
-Documentation=http://www.rsyslog.com/doc/
+Documentation=https://www.rsyslog.com/doc/
 
 [Service]
 Type=notify
@@ -18,4 +16,4 @@ LimitNOFILE=16384
 
 [Install]
 WantedBy=multi-user.target
-;Alias=syslog.service
+Alias=syslog.service
